Formula/qt.rb

```diff
@@ -89,13 +124,17 @@ def install
       -DTEST_assimp=ON
     ]
 
+    # Explicitly specify QT_BUILD_INTERNALS_RELOCATABLE_INSTALL_PREFIX so
+    # that cmake does not think $HOMEBREW_PREFIX/lib is the install prefix.
+    cmake_args << "-DQT_BUILD_INTERNALS_RELOCATABLE_INSTALL_PREFIX=#{prefix}" if OS.linux?
+
     system "./configure", *config_args, "--", *cmake_args
     system "cmake", "--build", "."
     system "cmake", "--install", "."
 
     rm bin/"qt-cmake-private-install.cmake"
 
-    inreplace lib/"cmake/Qt6/qt.toolchain.cmake", Superenv.shims_path, "/usr/bin"
+    inreplace lib/"cmake/Qt6/qt.toolchain.cmake", Superenv.shims_path, ""
```

Comment:
I think it's preferable here to just remove `Superenv.shims_path`, rather than hardcode `/usr/bin`.